### DISCUSSION TITLE
Show all data - Boolean fields - Unknown Value

### DISCUSSION THREAD

**Discussion by BTJason** [tprouvot/Salesforce-Inspector-reloaded#374] [Q&A] [answered]
What is meant by the value of "unknown" for boolean fields, is that because of the nature of what False really means when it comes to boolean?

**Comment by tprouvot** [comment, marked as answer]
Hi @BTJason ,

That's because you don't have the read right on this field. 

Add the rights to one of your pset or profile and you'll see the value.

**Comment by BTJason** [comment]
Thank you, Thomas, and here I am getting philosophical on the nature of boolean fields 😄


Jason Rojas

Salesforce Administrator





 [cid:9c2a89d2-b78b-463e-bd6a-9f9aeae59f8a]
